<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineValuesWidget</class>
 <widget class="QWidget" name="LineValuesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Line Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Region</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="_regionComboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Line</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="_linesComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTableWidget" name="_lineTableWidget">
      <property name="columnCount">
       <number>6</number>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QTableWidget" name="_regionsTableWidget">
      <property name="columnCount">
       <number>4</number>
      </property>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>_regionComboBox</sender>
   <signal>activated(int)</signal>
   <receiver>LineValuesWidget</receiver>
   <slot>region_chosen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_linesComboBox</sender>
   <signal>activated(int)</signal>
   <receiver>LineValuesWidget</receiver>
   <slot>line_chosen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
